test/unit/org/apache/cassandra/concurrent/SEPExecutorTest.java

```diff
@@ -174,9 +191,9 @@ void assertMaxTaskConcurrency(LocalAwareExecutorService executor, int concurrenc
         CountDownLatch concurrencyGoal = new CountDownLatch(concurrency);
         for (int i = 0; i < concurrency; i++)
         {
-            executor.execute(new LatchWaiter(concurrencyGoal, 5L, TimeUnit.SECONDS));
+            executor.execute(new LatchWaiter(concurrencyGoal, 5000L, TimeUnit.MILLISECONDS));
         }
         // Will return true if all of the LatchWaiters count down before the timeout
-        Assert.assertEquals(true, concurrencyGoal.await(3L, TimeUnit.SECONDS));
+        Assert.assertEquals(true, concurrencyGoal.await(3000L, TimeUnit.MILLISECONDS));
```

Comment:
is there a reason you changed the units?  the values are the same so not sure if there is a reason.

Reply:
while you are here, can you provide a more meaningful error message? =D

Reply:
leftover cruft from local reproduction - will revert and improve message.